### DISCUSSION TITLE
Why is high cfg increasing time of image generation?

### DISCUSSION THREAD

**Discussion by okolenmi** [Comfy-Org/ComfyUI#2081] [Q&A] [answered]
I'm running SDXL models with LCM. As recomended, I set cfg to 1 and time to iterate 1 step is 12seconds. When I trying to change cfg to 1.5 it becomes exactly 2 times slower.

![изображение](https://github.com/comfyanonymous/ComfyUI/assets/55319814/fed0f97b-ccf4-42a2-a6a8-b9e2388e3289)

Why is it so? Tested it several times. Even after ComfyUI reloading I have the same results.

**Comment by comfyanonymous** [comment, marked as answer]
cfg 1.0 means the negative prompt is ignored so only the positive noise is predicted which is why it's 2x faster.

**Reply by okolenmi** [reply]
Is it possible to ignore negative prompt with high cfg? I have almost nothing in negative.

**Reply by okolenmi** [reply]
Oh, empty negative prompt is around 30% slower. 15 seconds. But even this is good, I think. Thanks.
